src/components/DockApp.vue

```diff
@@ -188,26 +248,58 @@ export default {
   }
 }
 
+@keyframes bounce-horizontal-reverse {
+  0%,
+  100% {
+    transform: translate3d(0, 0, 0);
+  }
+  50% {
+    transform: translate3d(-60%, 0, 0);
+  }
+  90% {
+    transform: translate3d(5px, 0, 0);
+  }
+}
+
 // enable icon hover animation and 
 // name visibility on desktops only
 
 @media (hover: hover) and (pointer: fine) {
   .dock-app-container {
     &:hover {
-      margin: 0px 15px;
-      .dock-app-name  {
+      .dock-app-icon-container {
+        .dock-app-notification {
+          transform: scale(1.3) translate(-1px, -17px);
+        }
+      }
+
+      .dock-app-icon,
+      svg {
+        width: 78px;
+        height: 78px;
+        margin-top: -20px;
+        transform: translateY(-5px);
+      }
+      .dock-app-name {
         visibility: visible;
       }
     }
-    .dock-app-icon-container:hover {
-      transform: scale3d(1.3, 1.3, 1.3) translate3d(0, -10px, 0);
-    }
     &.dock-app-container-left {
       &:hover {
-        margin: 15px 0;
-      }
-      .dock-app-icon-container:hover {
-        transform: scale3d(1.3, 1.3, 1.3) translate3d(10px, 0, 0);
+        .dock-app-icon-container {
+          .dock-app-notification {
+            transform: scale(1.3) translate(20px, 1px);
```

Comment:
And use `scale3d` and `translate3d` here as well for GPU acceleration?

Reply:
Done